docs/components/concepts/job-workers.md

```diff
@@ -84,7 +82,7 @@ When `Completing or failing jobs` with [variables](components/concepts/variables
 There are several advantages when failing a job with variables. Consider the following use cases:
 
 - You can fail a job and raise an incident by setting the job `retries` to zero. In this case, it would be useful to provide some additional details through a variable when the incident is analyzed.
-- If your job worker can split the job into smaller pieces and finish some but not all of these, it can fail the job with variables indicating which parts of the job were successfully finished and which weren't. Such a job should be failed with a positive number of retries so another job worker can pick it up again and continue where the other job worker left off. The job can be completed when all parts are finished by a job worker successfully.
+- If your job worker can split the job into smaller pieces and finish some but not all of these, it can fail the job with variables indicating which parts of the job were successfully finished which weren't. Such a job should be failed with a positive number of retries so another job worker can pick it up again and continue where the other job worker left off. The job can be completed when all parts are finished by a job worker successfully.
 
 :::
 
```

Comment:
Do we need to update the job streaming note below?

Reply:
@npepinpe can you clarify?

Reply:
Nudging @npepinpe here, but can always merge this in and launch a separate PR.

Reply:
No, job streaming just affects how the job gets to your worker, but not the semantics of the worker itself. This is still correct :+1: 

Sorry for the slow reply :s